### ISSUE TITLE
Add confidence scoring to distillation process

### ISSUE THREAD

**Issue by Floppy** [theodi/shared#503]
Let database model confidence, and let ETLs submit confidence through the API. This is our understanding of what we need to do for OpenAddressesUK/roadmap#27

**Comment by giacecco** [comment]
Blocked waiting for Fusion Data Science to be back.

**Comment by giacecco** [comment]
@floppy @pezholio it is more useful if we move this straight to next sprint, to give Fusion time to be back and give us the theory first. The two points will be used to cover for the additions. Is that ok for you?

**Comment by giacecco** [comment]
Discussed during standup, no objections, this becomes a candidate for sprint 41.

**Comment by giacecco** [comment]
Changed title to be closer to what will actually happen.

**Comment by giacecco** [comment]
Specifications are being written at https://drive.google.com/open?id=1955JNRhDGMckIRObgS-9cEduY5p6cX6fKMjY-peSE8Q&authuser=0 . More work is needed before we can proceed to implementation. It is likely that the document will be completed by 11/2 and we will use @murraydata's time in the office on 12/2 to discuss and start thinking to how to implement.

**Comment by giacecco** [comment]
@floppy, @pezholio the confidence scoring specifications are [complete](https://docs.google.com/document/d/1955JNRhDGMckIRObgS-9cEduY5p6cX6fKMjY-peSE8Q/edit?usp=sharing). We presented them quickly to @pezholio this afternoon but we need you to study the document from top to bottom with a fresh mind so that you can understand any of its implications on implementation.

@murraydata is available to help for the more math-y content.

@peterkwells @jenit the document is ready to be published, too. It probably makes sense to reference it from the confidence blog post scheduled for next week rather than advertise it directly. Google Docs is configured so that anybody with the link can comment.

**Comment by giacecco** [comment]
Changed to low priority. After today's catch-up with the team this is unlikely to fit the current sprint and I need not to dilute the remaining time and effort. FYI @peterkwells @jenit.

**Comment by JeniT** [comment]
This will need to build on #546

**Comment by giacecco** [comment]
Duplicate of #504
